moped-editor/src/views/projects/projectsListView/ProjectsListViewMap.js

```diff
@@ -49,49 +99,64 @@ const ProjectsListViewMap = ({
   }, [loading, setIsMapDataLoading]);
 
   /* Build feature collection to pass to the map, add project_geography attributes to feature properties along with status color */
-  const projectsFeatureCollection = React.useMemo(() => {
-    const projectGeographiesFeatureCollection =
-      projectsGeographies?.project_geography
-        ? projectsGeographies?.project_geography.reduce(
-            (acc, projectGeography) => {
-              const phaseKey =
-                projectDataById[projectGeography.project_id]?.current_phase_key;
-              const statusBadgeColor = phaseKey
-                ? styleMapping[phaseKey]?.background
-                : styleMapping.default.background;
-
-              const projectGeographyFeature = {
-                type: "Feature",
-                geometry: projectGeography.geography,
-                properties: {
-                  ...(projectGeography.attributes
-                    ? projectGeography.attributes
-                    : {}),
-                  color: statusBadgeColor,
-                },
-              };
-
-              return {
-                ...acc,
-                features: [...acc.features, projectGeographyFeature],
-              };
-            },
-            { type: "FeatureCollection", features: [] }
-          )
-        : { type: "FeatureCollection", features: [] };
-
-    return projectGeographiesFeatureCollection;
-  }, [projectsGeographies, projectDataById]);
+  const {
+    projectGeographiesFeatureCollectionLines,
+    projectGeographiesFeatureCollectionPoints,
+    featuredProjectsFeatureCollectionLines,
+    featuredProjectsFeatureCollectionPoints,
+  } = useProjectGeographies({
+    featuredProjectIds,
+    projectsGeographies,
+    projectDataById,
+  });
 
   return (
-    <>
+    <Paper component="main" className={classes.content}>
+      <MapDrawer title={"Projects"} ref={mapRef} open={open} setOpen={setOpen}>
+        <List>
+          {featuredProjectsData.length > 0 ? (
+            featuredProjectsData.map((projectData) => (
+              <ListItem key={projectData?.project_id} disablePadding>
+                <ListItemText
+                  primary={
+                    <Link
+                      component={RouterLink}
+                      to={`/moped/projects/${projectData?.project_id}`}
+                      state={{ queryString }}
+                    >
+                      {projectData?.project_name_full}
+                    </Link>
+                  }
+                  secondary={`#${projectData?.project_id}`}
+                />
+              </ListItem>
+            ))
+          ) : (
+            <ListItem disablePadding>
+              <ListItemText primary="No projects selected" />
```

Comment:
Thinking about user experience, what do you think about having this give direction instead, like "Select a project component to see corresponding project IDs"

its a little wordy, but opening the map with no context, I am not sure where to "select a project"

Reply:
"Select on map to see project IDs" ?

Reply:
Another option would be for there to be a banner at the top of the map or something like that when you first open it with the directive, that way a user doesnt have to open the sidebar to see the directive, kinda like how the project map tells you to zoom in to draw features with a banner at the top.

Reply:
y'all - i love these ideas and i'll update to say **Select components on map to list projects** - open to more feedback on that too.

@roseeichelmann I spun https://github.com/cityofaustin/atd-data-tech/issues/17687 out of one of Chia's PRs earlier. There is definite overlap with your idea so please edit or comment in that issue - i'd really like to capture your thoughts about how this could look in the future. thanks!